react-app/App.css

```diff
@@ -0,0 +1,9 @@
+.test-div {
```

Comment:
Choose a more appropriate name for this class

Reply:
Done!